subsys/net/lib/nrf_cloud/coap/src/coap_codec.c

```diff
@@ -292,8 +292,35 @@ static void copy_wifi_info(struct wifi_ob *wifi_encode,
 	}
 }
 
+static bool copy_config(struct config *config_encode,
+			struct nrf_cloud_location_config const *const conf)
+{
+	bool needed = false;
+
+	/** Check each field and compare to the cloud's default. If different,
+	 *  enable output of the field and set the new value of it.
+	 */
+	if (conf->do_reply != true) {
```

Comment:
can you make defines for the cloud default values?
to be used here and in `nrf_cloud_obj_location_request_create` and in `nrf_cloud_rest_location_get`